ee_tests/oc-get-jenkins-logs.sh

```diff
@@ -32,4 +32,4 @@ echo --- Events in Jenkins project ---
 oc get ev
 
 echo --- Jenkins pod log ---
-oc logs `oc get pods -o name | grep jenkins -m 1`
+oc get pods -o name | xargs -L1 oc logs || true
```

Comment:
Is there a possibility for the `jenkins-*` pod not to appear at a first line of `oc get pods`?
If that was the case, the Jenkins pod log would be missed completely. WDYT about the following?

```bash
oc get pods -o name | grep jenkins | xargs -L1 oc logs || true
```

Reply:
I think it is not necessary since xargs will run oc logs command for all pods listed by oc get pods command. But it might be possible that we would get also logs for some other pods. I am not sure if that is good or bad though :-)

Reply:
Ah, you're right - it gets logs for all the pods there, not just Jenkins. The comment `--- Jenkins pod log ---` above confused me :).

I suggest to change it to something like: `--- Jenkins project pod logs ---`.

If the purpose of getting the logs is to debug possible issues, IMHO it's useful to have as much data as possible. :)